aci/resource_aci_fvrsconsif.go

```diff
@@ -24,12 +24,20 @@ func resourceAciContractInterfaceRelationship() *schema.Resource {
 		},
 
 		SchemaVersion: 1,
-		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
```

Comment:
Why do we remove this function and add the annotation as separate entries to the Schema?

Reply:
This Function appends description and annotation attribute to the respective resource schema. Some of the ACI classes doesn't support description but may support annotation, Hence addition of annotation separately.

Reply:
Did you consider creating separate function for annotation only and re-use this? Also have a look at the AppendAttrSchemas function in https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/aci/base_attr_schema.go, this could perhaps be leveraged.

Reply:
Added a GetAnnotation function for uniform schema implementation.